```
Fetching full git repo https://github.com/Cuperino/QPrompt.git
**fatal**: ambiguous argument '2.0.0-flatpak': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
Failed to download sources: module qprompt: Child process exited with code 128
error: Recipe `download-sources` failed with exit code 1
Error: Process completed with exit code 1.
```

--- a/com.cuperino.qprompt.json
+++ b/com.cuperino.qprompt.json
@@ -1,7 +1,7 @@
 {
    "id": "com.cuperino.qprompt",
    "runtime": "org.kde.Platform",
-   "runtime-version": "6.10",
+   "runtime-version": "6.9",
    "sdk": "org.kde.Sdk",
    "command": "qprompt",
    "finish-args": [

--- a/com.cuperino.qprompt.json
+++ b/com.cuperino.qprompt.json
@@ -1,7 +1,7 @@
 {
    "id": "com.cuperino.qprompt",
    "runtime": "org.kde.Platform",
-   "runtime-version": "6.9",
+   "runtime-version": "6.10",
    "sdk": "org.kde.Sdk",
    "command": "qprompt",
    "finish-args": [
@@ -53,8 +53,8 @@
             {
                "type": "git",
                "url": "https://github.com/Cuperino/QPrompt.git",
-               "tag": "2.0.0-flatpak",
-               "commit": "1db51855c1d58cae43bec1e019682d55b483b610"
+               "tag": "v2.0.0-flatpak",
+               "commit": "5632526c609ef9b0d5f1a63b06249c7a2d240043"
             }
          ]
       }

--- a/com.cuperino.qprompt.json
+++ b/com.cuperino.qprompt.json
@@ -1,7 +1,7 @@
 {
    "id": "com.cuperino.qprompt",
    "runtime": "org.kde.Platform",
-   "runtime-version": "6.9",
+   "runtime-version": "6.8",
    "sdk": "org.kde.Sdk",
    "command": "qprompt",
    "finish-args": [

